lib/utils/isStandardSyntaxDeclaration.js

```diff
@@ -36,6 +37,11 @@ module.exports = function (decl) {
 		return false;
 	}
 
+	// SCSS map and list declarations
+	if (value[0] === '(' && value[value.length - 1] === ')') {
```

Comment:
Let's use string functions for this, which we've moved to in the other `isStandardSyntax*` utils:

```suggestion
	if (value.startsWith('(') && value.endsWith(')')) {
```